### PR TITLE
Update to GNOME 3.30 & update source to gitlab instance

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: org.gnome.Hitori.json [hunk 1]
```diff
@@ -16,11 +16,6 @@
       "--talk-name=ca.desrt.dconf",
       "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
     ],
-  "build-options" :
-    {
-      "cflags": "-O2 -g",
-      "cxxflags": "-O2 -g"
-    },
```
Comment (pwithnall) [inline org.gnome.Hitori.json]: The commit message doesn’t mention why the `build-options` have been removed.

Comment (TingPing) [reply org.gnome.Hitori.json]: `flatpak-builder` parses this file which is now in the runtime: `flatpak run --command=cat org.gnome.Sdk//3.30 /etc/flatpak-builder/defaults.json`

Comment (pwithnall) [reply org.gnome.Hitori.json]: Nice, thanks for the explanation. This looks good to merge once that’s mentioned in the commit message.

**Comment by pwithnall** [comment]
…or we could just merge it without modifying the commit message.

**Pull request merged**